packages/core/src/components/slider/handleProps.tsx

```diff
@@ -47,6 +47,8 @@ export const HandleInteractionKind = {
 // eslint-disable-next-line @typescript-eslint/no-redeclare
 export type HandleInteractionKind = typeof HandleInteractionKind[keyof typeof HandleInteractionKind];
 
+export type HandleHtmlProps = Pick<React.HTMLProps<HTMLSpanElement>, "aria-label" | "aria-labelledby">;
```

Comment:
do we actually want to `Pick` these specific props or should we just let any `span` props be passed? (Should this just be `React.HTMLProps<HTMLSpanElement>`?)

Reply:
sounds good